.circleci/config.yml

```diff
@@ -646,21 +655,47 @@ jobs:
             - run:
                 name: Ensure package.json Version And Tag Match
                 command: cd desktop/bin && node validate_tag.js $VERSION
-      - run: *update-node
-      - run: *yarn-install
+      - run:
+          name: Install Node
+          command: |
+            set +e
+            set +x
+            export NVM_DIR="/opt/circleci/.nvm"
```

Comment:
Unfortunately the Linux machine executor is not without it's own set of unique bugs. The `NVM_DIR` has to be refreshed in a very specific way (see [this thread](https://discuss.circleci.com/t/circleci-forgetting-node-version-on-machine-executor/28813)). Further, I tried consolidating the use of `export NVM_DIR="/opt/circleci/.nvm"` into a separate shell script as in [this](https://github.com/ClouDesire/docusaurus/commit/ee11951336be3f64f5c6a25a8047bfeecc29fb43) example, but this didn't work (would result in `command nvm not found` error).

Reply:
You could as well just merge all commands into one big command, there is not a lot of benefit in having them as separate `run` commands.

Reply:
True - I can probably refactor in a downstream PR as there's a few enhancements to this I'd like to test (also better dependency caching).